portal/src/main/webapp/js/src/dashboard/hijackSubmission.js

```diff
@@ -0,0 +1,47 @@
+
+function handleLegacySubmission(url, query){
```

Comment:
@zhx828 these are dirty tricks to hijack submission of queries to results page and patient page from legacy study view.   We we localStorage to avoid having to post giant queries to the SPA(which would require backend support).    Obviously all this goes away when we introduce new study view, although we may actually need the localStorage trick if we want app to open in new window.